package.json

```diff
@@ -34,6 +34,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "eslint": "^6.6.0",
-    "spawn-command": "0.0.2-1"
+    "spawn-command": "0.0.2-1",
+    "split2": "3.1.1"
```

Comment:
This is required because graph-cli is missing this required dependency in their package.json. Without this, a run-time error occurs when graph-cli codegen is called when running the release.sh script.